index.js

```diff
@@ -807,8 +821,23 @@ RingPop.prototype.handleOrProxy =
         if (this.whoami() === dest) {
             return true;
         } else {
-            this.proxyReq(dest, req, res, opts);
+            this.proxyReq(_.extend(opts, {
```

Comment:
yes, definitely. for some reason i thought it wouldn't mutate `opts` and would generate a copy.